File: my-folder/0001-two-sum/solution.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] indices = new int[2];
        
        outerloop:
        for(int i = 0; i < nums.length; i++){
            indices[0] = i;
            for(int j = i + 1; j < nums.length; j++){
                if((nums[i] + nums[j]) == target){
                    indices[1] = j;
                    break outerloop;
                }
            }
        }
        
        return indices;
    }
}

File: my-folder/0001-two-sum/solution.rb
# @param {Integer[]} nums
# @param {Integer} target
# @return {Integer[]}
def two_sum(nums, target)
    sums = {}
    nums.each_with_index do |num, index|
        diff = target - num
        if sums.has_key?(diff)
            return [sums[diff], index]
        end
        sums[num] = index
    end
end
